resolver/oneInchmulti.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+export async function ExecuteQuote(quoteParams) {
+  for (var i = 0; i < quoteParams.fromTokenAddressArray.length; i++) {
+    var singlequoteParams = {
```

Comment:
Let's use `let` and `const` instead for var, as we are using ES6js

Reply:
Oh, I did not know that about ES6. Just read about it and made the change. Thanks